rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md

```diff
@@ -0,0 +1,41 @@
MDhWNDAuNDAwOFoiIGZpbGw9IiMyOUFCRTIiLz4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0zMTkuNiAxNjkuNDAyQzMxNC42IDEwMC40MDIgMjU5LjQgNDUuMjAxNiAxOTAuNCA0MC4yMDE2VjI0LjYwMTZDMjY4IDI5LjgwMTYgMzMwLjIgOTEuODAxNiAzMzUuMiAxNjkuNDAySDMxOS42WiIgZmlsbD0idXJsKCNwYWludDFfbGluZWFyXzk4MF8xMzEpIi8+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTkwLjQgMzE5LjYwMkMyNTkuNCAzMTQuNjAyIDMxNC42IDI1OS40MDIgMzE5LjYgMTkwLjQwMkgzMzUuMkMzMzAuMiAyNjguMDAyIDI2OCAzMzAuMDAyIDE5MC40IDMzNS4yMDJWMzE5LjYwMloiIGZpbGw9IiMyOUFCRTIiLz4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0xODAuMTkgMTkxLjA3OUMyMDYuODQ2IDE5MS4wNzkgMjI5LjEyMyAxODYuNTcyIDIzNC41NjggMTgwLjU1QzIyOS45NDUgMTc1LjQ0MyAyMTMuMjI0IDE3MS40MjEgMTkxLjk4OSAxNzAuMzJWMTgzLjA0MkMxODguMTg4IDE4My4yNCAxODQuMjM1IDE4My4zMzYgMTgwLjE4NiAxODMuMzM2QzE3Ni4xMzcgMTgzLjMzNiAxNzIuMTg0IDE4My4yNCAxNjguMzc1IDE4My4wNDJWMTcwLjMyQzE0Ny4xNDggMTcxLjQyMSAxMzAuNDE5IDE3NS40NDMgMTI1Ljc5NyAxODAuNTVDMTMxLjI0OSAxODYuNTcyIDE1My41MyAxOTEuMDc5IDE4MC4xODYgMTkxLjA3OUgxODAuMTlaTTIyNy41NSAxMzEuMTk4VjE0OC43MTdIMTkxLjk4OVYxNjAuODY1QzIxNi45NjggMTYyLjE2MyAyMzUuNzExIDE2Ny41MDMgMjM1Ljg1IDE3My44OTNWMTg3LjIxNUMyMzUuNzExIDE5My42MDUgMjE2Ljk2OCAxOTguOTMzIDE5MS45ODkgMjAwLjIzNVYyMzAuMDVIMTY4LjM3OVYyMDAuMjM1QzE0My40MDEgMTk4LjkzNyAxMjQuNjY1IDE5My42MDUgMTI0LjUyNiAxODcuMjE1VjE3My44OTNDMTI0LjY2NSAxNjcuNTAzIDE0My40MDEgMTYyLjE2MyAxNjguMzc5IDE2MC44NjVWMTQ4LjcxN0gxMzIuODE4VjEzMS4xOThIMjI3LjU1NEgyMjcuNTVaTTExOC44MzMgMTAzLjMxOEgyNDMuNjdDMjQ2LjY1NCAxMDMuMzE4IDI0OS40MDEgMTA0Ljg4NyAyNTAuODg5IDEwNy40MzdMMjg3LjI1NiAxNjkuODg2QzI4OS4xMzkgMTczLjEyNiAyODguNTgxIDE3Ny4yMTQgMjg1Ljg5MiAxNzkuODM3TDE4NS43ODUgMjc3LjU1N0MxODIuNTM4IDI4MC43MjMgMTc3LjMyMiAyODAuNzIzIDE3NC4wODMgMjc3LjU1N0w3NC4xMDAyIDE3OS45NjlDNzEuMzUyOCAxNzcuMjggNzAuODMzNiAxNzMuMDc1IDcyLjg2MDIgMTY5LjgyTDExMS43MzggMTA3LjI0M0MxMTMuMjUzIDEwNC44MSAxMTUuOTQyIDEwMy4zMjEgMTE4LjgzNyAxMDMuMzIxTDExOC44MzMgMTAzLjMxOFoiIGZpbGw9InVybCgjcGFpbnQyX2xpbmVhcl85ODBfMTMxKSIvPgo8L2c+CjxkZWZzPgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MF9saW5lYXJfOTgwXzEzMSIgeDE9IjEzMC43MiIgeTE9IjMwNC4xMjIiIHgyPSIzMy40OCIgeTI9IjIyMi4yMjIiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KPHN0b3Agb2Zmc2V0PSIwLjIxIiBzdG9wLWNvbG9yPSIjRUQxRTc5Ii8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iIzUyMjc4NSIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MV9saW5lYXJfOTgwXzEzMSIgeDE9IjMwOS4zMiIgeTE9IjEyMy4wNjIiIHgyPSIyMTIuMDgiIHkyPSI0MS4xNjE1IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIG9mZnNldD0iMC4yMSIgc3RvcC1jb2xvcj0iI0YxNUEyNCIvPgo8c3RvcCBvZmZzZXQ9IjAuNjgiIHN0b3AtY29sb3I9IiNGQkIwM0IiLz4KPC9saW5lYXJHcmFkaWVudD4KPGxpbmVhckdyYWRpZW50IGlkPSJwYWludDJfbGluZWFyXzk4MF8xMzEiIHgxPSIzMjMiIHkxPSIzMzkuNSIgeDI9IjIxNy4zNjkiIHkyPSIxNTUuNTQyIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiNGMTVBMjQiLz4KPHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjRkJCMDNCIi8+CjwvbGluZWFyR3JhZGllbnQ+CjxjbGlwUGF0aCBpZD0iY2xpcDBfOTgwXzEzMSI+CjxyZWN0IHdpZHRoPSIzNjAiIGhlaWdodD0iMzYwIiBmaWxsPSJ3aGl0ZSIvPgo8L2NsaXBQYXRoPgo8L2RlZnM+Cjwvc3ZnPgo="; }}})'
```

Comment:
`4449444c0e6b03ba9edbce040186dac7c60906c3cab9d20a0b6c05b59798d00202e898cad20405e9c2d6a70905a1eedde50a0598c6c2cf0b056e036c04e7aba21404cdd1f4e20404e4d8cce80b0494e2c9d00c046e7d6e716c03b59798d00207ffb08aab0809cb83d3c90f0a6e086c04e7aba2147dcdd1f4e2047de4d8cce80b7d94e2c9d00c7d6d686e686c0292bec7bd030cfdb5c9a9070d6c0299e7a4cf087db4e3ade809716c05c295a993017b9efeb9a40371f2c794ae037db1a38bf40d7191c9aafe0d71010002012a3078363837343936363546463844326431313246613835394141323933463037413632323738324633380606636b58415554018e1e646174613a696d6167652f7376672b786d6c3b6261736536342c50484e325a79423361575230614430694d7a59774969426f5a576c6e61485139496a4d324d434967646d6c6c64304a76654430694d43417749444d324d43417a4e6a416949475a7062477739496d3576626d5569494868746247357a50534a6f644852774f693876643364334c6e637a4c6d39795a7938794d4441774c334e325a79492b436a786e49474e7361584174634746306144306964584a734b434e6a62476c774d4638354f4442664d544d784b53492b436a78775958526f49475139496b30784f4441674d454d794e7a6b754e43417749444d324d4341344d43343249444d324d4341784f4442444d7a5977494449334f533430494449334f53343049444d324d4341784f4441674d7a5977517a67774c6a59674d7a5977494441674d6a63354c6a51674d4341784f4442444d4341344d433432494467774c6a59674d4341784f4441674d466f6949475a706247773949694d7a516a4177516a6b694c7a344b50484268644767675a6d6c73624331796457786c50534a6c646d56756232526b4969426a62476c774c584a3162475539496d56325a5735765a47516949475139496b30304d4334304d444178494445354d4334304d444a444e4455754e4441774d5341794e546b754e444179494445774d43343249444d784e4334324d4449674d5459354c6a59674d7a45354c6a59774d6c597a4d7a55754d6a4179517a6b794c6a41774d4445674d7a4d774c6a41774d69417a4d4341794e6a67754d444179494449304c6a67674d546b774c6a51774d6b67304d4334304d444178576949675a6d6c736244306964584a734b434e7759576c756444426662476c755a574679587a6b344d4638784d7a45704969382b436a78775958526f49475a7062477774636e56735a5430695a585a6c626d396b5a43496759327870634331796457786c50534a6c646d56756232526b4969426b50534a4e4d5459354c6a59674e4441754e4441774f454d784d4441754e6941304e5334304d444134494451314c6a51774d4445674d5441774c6a59774d5341304d4334304d444178494445324f5334324d4446494d6a51754f454d794f53343449446b794c6a41774d4467674f5449754d4441774d5341794f5334344d444134494445324f533432494449304c6a67774d4468574e4441754e4441774f466f6949475a706247773949694d794f554643525449694c7a344b50484268644767675a6d6c73624331796457786c50534a6c646d56756232526b4969426a62476c774c584a3162475539496d56325a5735765a47516949475139496b307a4d546b754e6941784e6a6b754e444179517a4d784e433432494445774d4334304d4449674d6a55354c6a51674e4455754d6a41784e6941784f5441754e4341304d4334794d444532566a49304c6a59774d545a444d6a5934494449354c6a67774d5459674d7a4d774c6a49674f5445754f4441784e69417a4d7a55754d6941784e6a6b754e44417953444d784f533432576949675a6d6c736244306964584a734b434e7759576c756444466662476c755a574679587a6b344d4638784d7a45704969382b436a78775958526f49475a7062477774636e56735a5430695a585a6c626d396b5a43496759327870634331796457786c50534a6c646d56756232526b4969426b50534a4e4d546b774c6a51674d7a45354c6a59774d6b4d794e546b754e43417a4d5451754e6a417949444d784e433432494449314f5334304d4449674d7a45354c6a59674d546b774c6a51774d6b677a4d7a55754d6b4d7a4d7a41754d6941794e6a67754d444179494449324f43417a4d7a41754d444179494445354d43343049444d7a4e5334794d444a574d7a45354c6a59774d6c6f6949475a706247773949694d794f554643525449694c7a344b50484268644767675a6d6c73624331796457786c50534a6c646d56756232526b4969426a62476c774c584a3162475539496d56325a5735765a47516949475139496b30784f4441754d546b674d546b784c6a41334f554d794d4459754f445132494445354d5334774e7a6b674d6a49354c6a45794d7941784f4459754e5463794944497a4e4334314e6a67674d5467774c6a5531517a49794f5334354e4455674d5463314c6a51304d7941794d544d754d6a4930494445334d5334304d6a45674d546b784c6a6b344f5341784e7a41754d7a4a574d54677a4c6a41304d6b4d784f4467754d546734494445344d7934794e4341784f4451754d6a4d31494445344d79347a4d7a59674d5467774c6a45344e6941784f444d754d7a4d32517a45334e6934784d7a63674d54677a4c6a4d7a4e6941784e7a49754d546730494445344d7934794e4341784e6a67754d7a6331494445344d7934774e444a574d5463774c6a4d79517a45304e7934784e4467674d5463784c6a51794d5341784d7a41754e444535494445334e5334304e444d674d5449314c6a63354e7941784f4441754e5456444d544d784c6a49304f5341784f4459754e546379494445314d7934314d7941784f5445754d446335494445344d4334784f4459674d546b784c6a41334f5567784f4441754d546c61545449794e7934314e5341784d7a45754d546b34566a45304f4334334d5464494d546b784c6a6b344f5659784e6a41754f445931517a49784e6934354e6a67674d5459794c6a45324d7941794d7a55754e7a4578494445324e7934314d444d674d6a4d314c6a6731494445334d7934344f544e574d5467334c6a49784e554d794d7a55754e7a4578494445354d7934324d4455674d6a45324c6a6b324f4341784f5467754f544d7a494445354d5334354f446b674d6a41774c6a497a4e5659794d7a41754d4456494d5459344c6a4d334f5659794d4441754d6a4d31517a45304d7934304d4445674d546b344c6a6b7a4e7941784d6a51754e6a5931494445354d7934324d4455674d5449304c6a55794e6941784f4463754d6a4531566a45334d7934344f544e444d5449304c6a59324e5341784e6a63754e54417a494445304d7934304d4445674d5459794c6a45324d7941784e6a67754d7a6335494445324d4334344e6a56574d5451344c6a63784e3067784d7a49754f444534566a457a4d5334784f5468494d6a49334c6a55314e4567794d6a63754e545661545445784f4334344d7a4d674d54417a4c6a4d784f4567794e444d754e6a64444d6a51324c6a59314e4341784d444d754d7a4534494449304f5334304d4445674d5441304c6a67344e7941794e5441754f446735494445774e7934304d7a644d4d6a67334c6a49314e6941784e6a6b754f446732517a49344f5334784d7a6b674d54637a4c6a45794e6941794f4467754e546778494445334e7934794d5451674d6a67314c6a67354d6941784e7a6b754f444d33544445344e5334334f4455674d6a63334c6a55314e304d784f4449754e544d34494449344d4334334d6a4d674d5463334c6a4d794d6941794f4441754e7a497a494445334e4334774f444d674d6a63334c6a55314e3077334e4334784d444179494445334f5334354e6a6c444e7a45754d7a55794f4341784e7a63754d6a67674e7a41754f444d7a4e6941784e7a4d754d446331494463794c6a67324d4449674d5459354c6a6779544445784d5334334d7a67674d5441334c6a49304d304d784d544d754d6a557a494445774e4334344d5341784d5455754f545179494445774d79347a4d6a45674d5445344c6a677a4e7941784d444d754d7a4978544445784f4334344d7a4d674d54417a4c6a4d784f466f6949475a7062477739496e56796243676a63474670626e517958327870626d5668636c38354f4442664d544d784b53497650676f384c32632b436a786b5a575a7a50676f3862476c755a57467952334a685a476c6c626e516761575139496e4268615735304d4639736157356c59584a664f546777587a457a4d53496765444539496a457a4d4334334d69496765544539496a4d774e4334784d6a4969494867795053497a4d7934304f43496765544939496a49794d6934794d6a496949476479595752705a5735305657357064484d39496e567a5a584a546347466a5a55397556584e6c496a344b50484e306233416762325a6d63325630505349774c6a49784969427a644739774c574e76624739795053496a52555178525463354969382b436a787a644739774947396d5a6e4e6c644430694d534967633352766343316a62327876636a3069497a55794d6a63344e53497650676f384c327870626d5668636b6479595752705a57353050676f3862476c755a57467952334a685a476c6c626e516761575139496e4268615735304d5639736157356c59584a664f546777587a457a4d53496765444539496a4d774f53347a4d69496765544539496a45794d7934774e6a496949486779505349794d5449754d44676949486b79505349304d5334784e6a45314969426e636d466b61575675644656756158527a50534a31633256795533426859325650626c567a5a53492b436a787a644739774947396d5a6e4e6c644430694d4334794d534967633352766343316a62327876636a3069493059784e5545794e43497650676f3863335276634342765a6d5a7a5a585139496a41754e6a676949484e3062334174593239736233493949694e47516b49774d3049694c7a344b504339736157356c59584a48636d466b615756756444344b50477870626d5668636b6479595752705a57353049476c6b50534a7759576c7564444a6662476c755a574679587a6b344d4638784d7a4569494867785053497a4d6a4d6949486b785053497a4d7a6b754e53496765444939496a49784e79347a4e6a6b6949486b79505349784e5455754e5451794969426e636d466b61575675644656756158527a50534a31633256795533426859325650626c567a5a53492b436a787a6447397749484e3062334174593239736233493949694e474d5456424d6a51694c7a344b50484e306233416762325a6d63325630505349784969427a644739774c574e76624739795053496a526b4a434d444e434969382b436a777662476c755a57467952334a685a476c6c626e512b436a786a62476c7755474630614342705a44306959327870634442664f546777587a457a4d53492b436a78795a574e30494864705a48526f5053497a4e6a41694947686c6157646f644430694d7a59774969426d6157787350534a3361476c305a53497650676f384c324e73615842515958526f50676f384c32526c5a6e4d2b436a777663335a6e50676f3d06636b58415554`

Reply:
arg encoding verified. Logo verified.

Reply:
contract address verified

Reply:
Everything looks good to me!